docker/dotcms/Dockerfile

```diff
@@ -14,71 +15,90 @@ ARG BUILD_ID=HEAD
 
 WORKDIR /srv
 
+# Set gradle options
+ENV GRADLE_USER_HOME=/build/src/core/dotCMS/.gradle
+ENV GRADLE_OPTS="-Dfile.encoding=utf-8 -Xmx2g"
+
+
 # dotCMS core distributed under GPLv3 license (https://github.com/dotCMS/core/blob/master/license.txt)
 COPY build-src/ /build/
 RUN chmod 500 /build/build_dotcms.sh && /build/build_dotcms.sh ${BUILD_FROM} ${BUILD_ID}
 
 RUN mkdir -p /srv/utils /srv/templates /srv/config /srv/home
 RUN chmod -R 666 /srv && find /srv/ -type d -exec chmod a+x {} \;
 
-##
-## Begin target image build
-##
-
-# OpenJDK distributed under GPLv2+Oracle Classpath Exception license (http://openjdk.java.net/legal/gplv2+ce.html)
-# Alpine Linux packages distributed under various licenses including GPL-3.0+ (https://pkgs.alpinelinux.org/packages)
-#ARG ARCH=arm64v8
-
-FROM alpine:3.13
+# ----------------------------------------------
```

Comment:
Since on the previous stage and the following is kind of DotCMS related, why not merge both or execute the git clone and call to `downloadDependencies clonePullTomcatDist`?

Reply:
Because git+ build makes the final image fat.  By building it in a completely different image, e.g. `dotcms-seed` and just copying the build over to the final image, we can have an image that does not have all the build and source control artifacts in it, making it much smaller.